components/brave_ads/browser/ads_tab_helper.h

```diff
@@ -12,7 +12,9 @@
 #include "base/macros.h"
 #include "base/memory/weak_ptr.h"
 #include "build/build_config.h"
+#if !defined(OS_ANDROID)
```

Comment:
see above

Reply:
Fixed in 1105be6d38f16175dba23eef73a248f35c2819e3